mypy_primer.py

```diff
@@ -1778,14 +1778,14 @@ def inner() -> int | None:
     Project(
         location="https://github.com/Avasam/Auto-Split",
         mypy_cmd="{mypy} src",
-        pip_cmd="{pip} install -r scripts/requirements.txt types-d3dshot types-keyboard"
-        " types-Pillow types-psutil types-PyAutoGUI types-pyinstaller types-pywin32 types-requests"
-        " types-toml",
+        pip_cmd="{pip} install certifi ImageHash numpy packaging PyQt6"
+        " types-d3dshot types-keyboard types-Pillow types-psutil types-PyAutoGUI types-pyinstaller"
+        " types-pywin32 types-requests types-toml",
     ),
     Project(
         location="https://github.com/Avasam/speedrun.com_global_scoreboard_webapp",
         mypy_cmd="{mypy} backend",
-        pip_cmd="{pip} install -r scripts/requirements.txt"
+        pip_cmd="{pip} install Flask PyJWT requests-cache sqlalchemy"
```

Comment:
SQLAlchemy dosn't have inline types (yet; they're adding them for the next major version bump) -- maybe `types-SQLAlchemy`?

```suggestion
        pip_cmd="{pip} install Flask PyJWT requests-cache types-SQLAlchemy"
```

Reply:
`types-SQLAlchemy` is required by `types-Flask-SQLAlchemy`. I'll just remove that entry.